api/hasura/actions/_handlers/syncCoSoul.ts

```diff
@@ -34,7 +34,7 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
 
     if (!tokenId) {
       // no tokenId on chain, lets clean up
-      await burned(address);
+      await burned(session.hasuraProfileId, address);
```

Comment:
id follow same argument pattern as below:

```suggestion
      await burned(address, session.hasuraProfileId);
```